services/libs/tinybird/pipes/active_days.pipe

```diff
@@ -73,7 +73,15 @@ SQL >
                 'discussion-comment',
                 'merge_request-opened',
                 'merge_request-comment',
-                'merge_request-review-approved'
+                'merge_request-review-approved',
+                'merge_request-review-changes-requested',
+                'merge_request-assigned',
+                'merge_request-review-requested',
+                'merge_request-merged',
+                'changeset-closed',
+                'changeset_comment-created',
+                'patchset_comment-created',
+                'patchset_approval-created'
```

Comment:
I believe the `patchset-created` is missing.
And that we should not include `merge_request-assigned` or `merge_request-review-requested` since they are not marked as contributions.